ui/src/app/base/components/DoughnutChart/DoughnutChart.tsx

```diff
@@ -0,0 +1,195 @@
+        cy={radius}
+        key={i}
+        onMouseOver={
+          tooltip
+            ? (event) => {
+                const { currentTarget } = event;
+                const bounding = event.currentTarget.getBoundingClientRect();
+                setShowTooltip(tooltip);
+                const updatePosition = (x: number, y: number) => {
+                  const html = document.querySelector("html");
+                  const offsetTop = html.scrollTop;
+                  const scrollLeft = html.scrollLeft;
+                  // Set the position of the tooltip, accounting for the width of the
+                  // segment and the page scroll.
+                  setTooltipStyle({
+                    // An additional space (10) is added to give some room from
+                    // the cursor.
+                    left: x - bounding.x + segmentWidth / 2 + 10 - scrollLeft,
+                    top: y - bounding.y + segmentWidth / 2 - offsetTop,
+                  });
+                };
+                // Set the initial position on mouse over.
+                updatePosition(event.clientX, event.clientY);
+                const mouseListener = (evt: MouseEvent) => {
+                  // Move the tooltip as the mouse moves.
+                  updatePosition(evt.pageX, evt.pageY);
+                };
+                currentTarget.addEventListener("mousemove", mouseListener);
+                // Store the remove listener.
+                removeListener.current = () => {
+                  currentTarget.removeEventListener("mousemove", mouseListener);
+                };
+              }
+            : null
+        }
+        onMouseOut={
+          tooltip
+            ? () => {
+                // Hide the tooltip and clean up the mouse move listener.
+                setShowTooltip(null);
+                setTooltipStyle(null);
+                removeListener.current();
+              }
+            : null
+        }
+        r={radius}
+        style={{
+          stroke: color,
+          strokeWidth: segmentWidth,
+          // The dash array used is:
+          // 1 - We want there to be a space before the first visible dash so
+          //     by setting this to 0 we can use the next dash for the space.
+          // 2 - This gap is the distance of all previous segments
+          //     so that the segment starts in the correct spot.
+          // 3 - A dash that is the length of the segment.
+          // 4 - A gap from the end of the segment to the start of the circle
+          //     so that the dash array doesn't repeat and be visible.
+          strokeDasharray: `0 ${startPosition.toFixed(
+            2
+          )} ${segmentLength.toFixed(2)} ${remainingSpace.toFixed(2)}`,
+        }}
+        // Rotate the segment so that the segments start at the top of
+        // the chart.
+        transform={`rotate(-90 ${radius},${radius})`}
+      />
+    );
+  });
+  return (
+    <div className="p-tooltip--right">
```

Comment:
I think a `className` prop would be good - then you could give it `"u-align--center"` which would make the NUMA view look like the design